guide/payments/receive.md

```diff
@@ -0,0 +1,215 @@
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/share.png"
+   retina = "/assets/images/guide/payments/receive/share@2x.png"
+   alt-text = "Screen showing the invoice with a share button"
+   caption = ""
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/share-sharesheet.png"
+   retina = "/assets/images/guide/payments/receive/share-sharesheet@2x.png"
+   alt-text = "Screen showing the invoice overlayed with a share sheet"
+   caption = ""
+   width = 250
+   height = 541
+%}
+
+</div>
+
+
+**Text**
+
+Sharing payment details as text is as straightforward as copying and pasting. It is also typical to provide the receiver with a button to conveniently share in other applications, with “share sheets,” which most mobile operating systems offer.
+
+**QR Codes**
+
+QR Codes are also a common way to exchange payment details when the two devices are near one another. It is not without its faults though, as some people may not be familiar with QR Codes, and lower-end phones may have difficulty reading them.
+
+Here are some constraints to consider when displaying QR Codes:
+
+- Complex QR Codes can be difficult for devices with low-resolution cameras to read.
+- Use high contrast between the QR Code blocks and their background.
+- Display the QR Code at the largest size, so the scanner can detect it easily.
+
+_For more technical details about optimizing   QR Codes [see this article](https://bitcoinops.org/en/bech32-sending-support/#creating-more-efficient-qr-codes-with-bech32-addresses)._
+
+## Awaiting the payment
+Once the payment details are shared, until the sender creates a transaction and broadcast it, there is uncertainty of when the payment would be completed. Think about how to keep the receiver informed while they are waiting.
+
+Similar to [the processes a sender]({{ '/guide/payments/send/#transaction-processing-and-confirmation' | relative_url }}) sees, with the inclusion of this waiting period, here is what the receiver can expect:
+
+[WHAT DOES THE RECEIVER SEE?]
+
+If your application has an area where the user can see a list of payment requests, this would be a good place to use it to also indicate which specific stage the payment is in.
+
+Then once the payment has been finalized, you should also consider what the receiver may want to do with those funds. You may want to help facilitate those follow-up activities e.g. moving the funds to a shared multi-key wallet or doing a [coinjoin]({{ '/guide/glossary/#coinjoin' | relative_url }}).
+
+
+## Address compatibility
+There are currently [three different versions]({{ '/guide/glossary/#address' | relative_url }}) of bitcoin addresses. From newest to oldest they are; Segwit, Script, and Legacy. It is the best practice for applications to support the newest address version, SegWit, as default.
```

Comment:
Add why it is the best practice

Reply:
we've probably got to add this to the holistic guide review so ill leave as is now, as its used in other places and make a note on #202